src/test/java/Pages/BasePage.java

```diff
@@ -0,0 +1,37 @@
+package Pages;
```

Comment:
A package in Java is used to group related classes. 

For example package "pages" - Includes the Page Objects (or Page Classes)

For example

![image](https://github.com/YuriiLyndiuk/A57-QA-Automation-Framework/assets/104761482/27e94b6d-4d57-4c2a-8022-e828ff1276f7)